### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by Tenischev**
**Description**

Update readme
- use CLI instead of old `ag`
- add bages
- update contributors list
- add licence

### CODE REVIEW

**Comment by Tenischev** [comment]
@derberg please, have a look

File: README.md [hunk 1]
```diff
@@ -60,67 +80,61 @@ In order for the generator to know what names to use for some parameters it's ne
...
-1. Install AsyncAPI Generator:
+1. Make required changes in the template.
+2. Run snapshot tests:
+   ```sh
+   npm test
+   ```
+   If there falling tests examine diff report and make an appropriate changes in template files or snapshots.
+1. Check output generation project. Install AsyncAPI Generator:
    ```
    npm install -g @asyncapi/generator
```
Comment (derberg) [inline README.md:120]:
```suggestion
   npm install -g @asyncapi/cli
```

**Review by derberg — COMMENTED** [review]
one tiny comment

**Review by derberg — APPROVED** [review]
👏🏼

**Pull request merged**